### PR TITLE
393 update README.md

### LINKED ISSUE #393: Update README.md

### PR DESCRIPTION
**Pull request by ZQUEMA**
This issue do a big update of the README.md file.
It a a new structure. It adds the lore and change the display of the authors.

It will be upgraded in #395 

It close #393

### CODE REVIEW

**Review by LucasTesnier — APPROVED** [review]

**Review by TomDUVAL-MAHE — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,26 +1,97 @@
...
+***
+
+## **For developers**
+
+### **Build the program**
 
 On Linux :
 
-You may use the build.sh or the following commands :
+- You may use the build.sh or the following commands :
```
Comment (31Nathan) [inline README.md:94]: Exec the scripts ?

File: README.md [hunk 2]
```diff
@@ -29,7 +100,7 @@ You may use the build.sh or the following commands :
 
 On Windows :
 
-* You may use the build.bat or the following commands :
+- You may use the build.bat or the following commands :
 
 ```bash
     mkdir build
```
Comment (31Nathan) [inline README.md:106]: Same

File: README.md [hunk 3]
```diff
@@ -38,11 +109,17 @@ On Windows :
...
 ```
 
-## Testing program
+### **Testing program**
+
+The tests are available on Linux.
+You can test the code using with the following commands.
+
 ```bash
     rm -rf ./build
```
Comment (31Nathan) [inline README.md:124]: Same

**Review by 31Nathan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ZQUEMA) [reply README.md:94]: done

File: README.md (hunk 2 above)
Comment (ZQUEMA) [reply README.md:106]: done

File: README.md (hunk 3 above)
Comment (ZQUEMA) [reply README.md:124]: done

**Review by LucasTesnier — APPROVED** [review]

**Review by 31Nathan — APPROVED** [review]

**Pull request merged**